app/models/participant_declaration.rb

```diff
@@ -150,6 +150,7 @@ def duplicate_declarations
       .where.not(user_id:, id:)
       .where(state: %w[submitted eligible payable paid])
       .where(
+        cpd_lead_provider:,
```

Comment:
does this not now change the behaviour and after a transfer lead provider B can be post for the same declarations and we end up double paying?